datafusion/substrait/Cargo.toml

```diff
@@ -46,6 +46,7 @@ url = { workspace = true }
 
 [dev-dependencies]
 datafusion = { workspace = true, features = ["nested_expressions"] }
+datafusion-catalog = { workspace = true }
```

Comment:
I think since datafusion re-exports `datafusion_catalog` we should be able to avoid this dependency:

https://github.com/apache/datafusion/blob/9ca6c2557488c1c608182542c1be96889b64fe29/datafusion/core/src/lib.rs#L731-L730